src/select.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import JWSelect from 'react-select';
+import classNames from 'classnames';
+import identity from './helpers/identity';
+
+export default class Select extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: props.defaultValue
+        };
+    }
+
+    /**
+     * Handle internal value change before continuing externally.
+     *
+     * @param {string} value the new value of the select
+     * @return {undefined}
+     */
+    onChange(value) {
+        value = (value === '') ? null : value;
+        this.setState({ value: value }, () => {
+            this.props.onChange(value);
+        });
+    }
+
+    /**
+     * Form interface method for getting the value of the input
+     *
+     * @return {(string|number)} the value of the select
+     */
+    getValue() {
+        return this.state.value;
+    }
+
+    getClasses() {
+        return classNames(
+            'select-wrapper',
+            {
+                error: this.props.errors.length,
+                disabled: this.props.disabled,
+                [`${this.props.className}`]: this.props.className
```

Comment:
I only wanted to supply the custom class if one is supplied (as apposed to defaulting to a default string and always supplying it). Example of this [here](https://github.com/JedWatson/classnames#dynamic-class-names-with-es2015).